EVENT.md

```diff
@@ -868,7 +868,7 @@ In the above example, the browser could have chosen to generate three reports:
 The browser may apply storage limits in order to prevent excessive resource
 usage.
 
-Strawman: There should be a limit of 1024 pending sources per source origin.
+Strawman: There should be a limit of 4096 pending sources per source origin.
```

Comment:
The spec allows individual implementations to choose their own value for this limit. Do we need to duplicate Chromium's choice here when it's already listed in https://github.com/WICG/attribution-reporting-api/blob/main/params/chromium-params.md?

Maybe we can start removing all the vendor-specific values from this document, since they easily get out of date and are in any case implementation-specific.

Reply:
+1 I am supportive of duplicating this. We can always link out to this doc.

Reply:
@csharrison just to confirm, you meant supportive of **not** duplicating and just linking to the params doc?

Reply:
Oops yes